openid-4-verifiable-presentations-1_0.md

```diff
@@ -215,6 +215,53 @@ Figure: Cross Device Flow
 
 (3) The Wallet prepares the Verifiable Presentation(s) of the Verifiable Credential(s) that the End-User has consented to. It then sends to the Verifier an Authorization Response where the Verifiable Presentation(s) are contained in the `vp_token` parameter.
 
+## Advanced Flow with Wallet Capabilities Discovery {#adv_flow}
+
+Below is a diagram of a flow where the End-User presents a Credential to a Verifier interacting with the End-User on a different device as the device the Wallet resides on. In this case, the Verifier starts the flow by discovering the Wallet's capabilities through an additional message exchange. This is especially useful, if the Verifier starts the process using a URL serving as alias for a group of wallets (e.g. a custom scheme) but would like to tailor the presentation request to the particular wallet. 
```

Comment:
"different device as the device the Wallet resides on" -> "different device than the device that the Wallet resides on" maybe reads more clearly (or maybe it's just me).

Reply:
I made a mistake, as the flow obviously is same device. 
The sentence (and with that the order of things) was copied from the other flows. It just spells out the fact that this is a same device flow. 
Changed the text to refer to the same device flow instead of duplicating text.

Reply:
please check